src/Native/Unix/System.Native/pal_io.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include "pal_types.h"
+#include "pal_errno.h"
```

Comment:
... pal_console.cpp now includes pal_io.h for the SystemNative_Pipe, but it doesn't include pal_errno.h so it gave a compile error.
The other cpp files including pal_io.h must have included pal_errno.h before it.